src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityReportListenerImpl.java

```diff
@@ -72,10 +72,11 @@ private void processEvent(@Nonnull DockerTraceabilityReport report) throws IOExc
                
         // Update containerInfo if available
         final InspectContainerResponse containerInfo = report.getContainer();
-        @CheckForNull Fingerprint containerFP = null;
+        final @CheckForNull Fingerprint containerFP;
```

Comment:
Currently we do not use this fingerprint outside, so there's no findbugs issue. I'll just move the definition to if statement